### DISCUSSION TITLE
Supporting Google Spanner Database with Superset

### DISCUSSION THREAD

**Discussion by arpan14** [apache/superset#23623] [Ideas]
Hi Team,

I was investigating the steps to support Spanner database in Superset.
https://cloud.google.com/spanner

I have read a few documentation that suggest integrating a new database is not very difficult in cases where there is a supported ORM layer. For spanner there is a compliant SQLAlchemy dialect which conforms to DBAPI driver.
* https://github.com/googleapis/python-spanner-sqlalchemy 
* https://github.com/googleapis/python-spanner/tree/main/google/cloud/spanner_dbapi 

Could you suggest any examples of other databases that have integrated recently (any documents, PRs) which can help us investigate and enable this?

**Comment by mdeshmu** [comment]
Following documentation should be helpful.

https://preset.io/blog/building-database-connector/

https://preset.io/blog/improving-apache-superset-integration-database-sqlalchemy/

**Comment by rusackas** [comment]
Thanks for the links @mdeshmu and thanks for the interest @arpan14.

Converting this to an "Ideas" discussion since it's more feature request than bug. We can discuss it further there, and then close it when you've solved it. We look forward to the PR!
